packages/core/src/components/Timeline/SimpleItem.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import IconRecord from '@airbnb/lunar-icons/src/interface/IconRecord';
+import useStyles from '../../hooks/useStyles';
+import { styleSheetSimpleItem } from './styles';
+import Row from '../Row';
+import useTheme from '../../hooks/useTheme';
+
+export type SimpleItemProps = {
+  /** Whether an item is the oldest (chronologically) in the collection of items. */
+  oldest?: boolean;
+
+  /** Content to render for the title of the item. */
+  children: NonNullable<React.ReactNode>;
+};
+
+/** Represents a visually deemphasized single item in a collection of chronologically items. */
+export default function SimpleItem({ oldest = false, children }: SimpleItemProps) {
```

Comment:
i think this should be consolidated into `Item` and the difference flagged by a prop called `secondary`

Reply:
it's easier to maintain the two together and helps the developer experience (using a flag instead of importing and rendering a different item secondary, deemphasized items). boolean flags for slight visual differences would be more consistent with lunar practices

Reply:
Agree it is more standard with lunar practices, I'll make that update